rm-deployment-status-enhancer-summary/vss-extension.json

```diff
@@ -0,0 +1,45 @@
+{
+	"manifestVersion": 1,
+	"id": "rm-deployment-status-enhancer-summary",
+	"version": "0.1.5",
+	"name": "RM deployment status - summary",
+	"description": "RM summary section extension sample",
+	"publisher": "openalm",
+	"links": {
+        "learn": {
+            "uri": "https://github.com/Microsoft/vso-extension-samples"
+        }
+    },
+	"targets": 
+	[
+		{
+			"id": "Microsoft.VisualStudio.Services"

```

Comment:
Works on-prem (update 2), too? If not, use "Microsoft.VisualStudio.Services.Cloud" instead